draft-ietf-radext-radiusdtls-bis.md

```diff
@@ -261,6 +261,10 @@ If implemented it MUST use the following rules:
 
 [^may-should-trustbase]: Open discussion: RFC6614 says "may" here. I think this should be a "should". There are some discussions to change this to "must". Input from TLS/UTA experts is appreciated.
 
+If implemented, RADIUS/(D)TLS clients SHOULD NOT be pre-configured with a list of known public CAs by the vendor or manufacturer. Instead, the clients SHOULD start off with an empty CA list. The addition of a CA SHOULD be done only when manually configured by an administrator.
+
+This scenario is the opposite of web browsers, where they are preconfigured with many known CAs. The goal there is security from third-party observers, but also the ability to communicate with any unknown site that presents a signed certificate. In contrast, the goal of RADIUS/DTLS is both security from third-party observers and the ability to communicate with only a small set of well-known servers.
+
```

Comment:
The idea is good, I completely agree.

I'm just not really comfortable with the text, I've tried to re-word it a bit, maybe you see where my problems are.
If you have an alternative: I'm happy to throw my re-wording away ;)

```suggestion
If this model is implemented, RADIUS/(D)TLS clients SHOULD NOT be pre-configured with a list of known public CAs by the vendor or manufacturer. Instead, the clients SHOULD start off with an empty CA list. The addition of a CA SHOULD be done only when manually configured by an administrator.

The goal of RADIUS/(D)TLS is to securely communicate with only a small set of well-known peers.
These peers will use specific certificates, potentially even from a private purpose-specific CA.
This scenario is different from a common use case of PKIX where an entity wants to communicate securely with unknown remote entities (i.e. web browsing), that use the public CAs to establish a trust path to this unknown entity.
```

Reply:
I am happy with these changes